Седьмая лабораторная работа выполнена

PopovMatvey opened the pull request:
1) Были полностью выполнены все задания с первого по четвёртое.

PopovMatvey committed:
Седьмая лабораторная работа выполнена

Были выполнены все задания с 1 по 4

PopovMatvey committed:
Create task-01.css

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Create SendFlowers-Regular.eot

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Create task-01.png

PopovMatvey committed:
Add files via upload

djilua1 commented on src/ex7_html-css-basics/css/task-01.css:
```diff
@@ -0,0 +1,85 @@
+.task {
```
Для первого задания не нужен css файл и стили.

djilua1 commented on src/ex7_html-css-basics/css/task-02.css:
```diff
@@ -0,0 +1,55 @@
+.description-block {
```
Для второго задания не нужен css файл и стили.

djilua1 commented on src/ex7_html-css-basics/task-02.html:
```diff
@@ -0,0 +1,180 @@
+
+        <p>
+            <span>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Iste omnis deserunt maxime. Perspiciatis
+                <br> </span>
```
br вне span.

djilua1 commented on src/ex7_html-css-basics/task-04.html:
```diff
@@ -0,0 +1,186 @@
+    </div>
+</body>
+
+</html>
```
![image](https://user-images.githubusercontent.com/43634050/161438133-2f4788ba-b02d-45ad-9dc4-b1643e251c55.png)

PopovMatvey committed:
Проверенные валидатором css файлы

PopovMatvey committed:
Delete task-01.css

PopovMatvey committed:
Delete task-02.css

PopovMatvey committed:
Изменённые css файлы

PopovMatvey committed:
Исправления по седьмой лабораторной работе

Были исправлены все 4 замечания:
1) Был удалён файл со стилями для первого задания и взамен стилям были добавлены соответствующие тэги атрибуты;
2) Был удалён файл со стилями для второго задания и взамен стилям были добавлены соответствующие тэги атрибуты;
3) В Первом, втором, четвёртом задании <br> был вынесен из <span>.
4) В четвёртом задании были использованы все существующие способы добавления стилей, такие как:
      1) Подключение css файла;
      2) Глобально,  с использованием тэга <style></style>;
      3) С использованием атрибута у элемента (style);
      4) Импорт css файла внутри тэга <style></style>;
      5) Импорт css файла внутри другого css файла (уже подключённого к странице).

PopovMatvey committed:
Update task-01.html

PopovMatvey committed:
Update task-02.html
